README.md

```diff
@@ -29,34 +29,40 @@ Sonobuoy supports 3 Kubernetes minor versions: the current release and 2 minor v
 * For some advanced workflows it may be required to have `kubectl` installed. See [installing via Homebrew (MacOS)][brew] or [building
   the binary (Linux)][linux].
 
+* The `sonobuoy images` subcommand requires [Docker](https://www.docker.com) installed. See [installing Docker](https://docs.docker.com/install).
```

Comment:
The other links in this file use reference style links with the links at the bottom of the file. Could you update this to use the same style here?

Also, could you modify this sentence to include "to be" before "installed".
```suggestion
* The `sonobuoy images` subcommand requires [Docker](https://www.docker.com) to be installed. See [installing Docker](https://docs.docker.com/install).
```

Reply:
Good catch, thanks! :)